lib_camera/api/camera_conv.h

```diff
@@ -119,50 +119,30 @@ void camera_int8_to_uint8(
  * @param channels  Number of channels
  */
 void camera_swap_dims(
-    uint8_t* image_in, 
-    uint8_t* image_out, 
-    const unsigned height, 
-    const unsigned width, 
+    uint8_t* image_in,
+    uint8_t* image_out,
+    const unsigned height,
+    const unsigned width,
     const unsigned channels);
 
 /**
- * @brief Rotates an RGB image 90 degrees clockwise 
- * assumes both src and dst images are previously allocated
- * Input image dimensions are [height][width][channel]
- * and output image dimensions are [width][height][channel]
- * Input image pointer can't be reused for the output image
- * 
- * @param dst_img   Destination image
- * @param src_img   Source image
- * @param h         Image height
- * @param w         Image width
- * @param ch        Number of channels
- */
-void camera_rotate90_uint8(
-    uint8_t* dst_img, 
-    uint8_t* src_img, 
-    const int16_t h, 
-    const int16_t w, 
-    const int16_t ch);
-
-/**
- * @brief Rotates an RGB image 90 degrees clockwise 
+ * @brief Rotates an RGB image 90 degrees clockwise
  * assumes both src and dst images are previously allocated
  * Input image dimensions are [height][width][channel]
  * and output image dimensions are [width][height][channel]
  * Input image pointer can't be reused for the output image
- * 
+ *
```

Comment:
could you add a comment avoid input type saying only int8 or uint8 are supported?

Reply:
I added some info in https://github.com/xmos/lib_camera/pull/91/commits/238f5c1788de81dd4580fc28babeabe64090bfab.
Is that ok?

Reply:
good!